update rpm page (#3771)
ArduPilot/ardupilot_wiki: common/source/docs/common-rpm.rst

For a reliable RPM signal the hall effect pick-up should be mounted very close to the magnet.  The image below shows an example mounting on a helicopter.

.. image:: ../../../images/Installed_Hall_Effect_Sensor.jpg
    :target: ../_images/Installed_Hall_Effect_Sensor.jpg
    :width: 240px

The three wires should then be plugged into the autopilot.  This is commonly done using a standard servo plug.  Ground to Gnd, Vcc to 5V, and the signal line 
needs to be attached to a GPIO pin.  On most smaller boards this will be any one of the PWM pins on the servo rail.  On a Pixhawk this must be one of the AUX ports.  
For reasons explained later it is recommended to use the highest number pin first.  E.g. AUX 6 on a Pixhawk/Cube or PWM 9 on an F405-Wing.

**Parameter Setup**

First the board needs to be configured to allow PWM pins to be set for GPIO.  This is done using the parameter ``BRD_PWM_COUNT`` .  Reduce the PWM count to free up a pin to 
be used for GPIO.  On non-Pixhawk boards the PWM count will include all PWM outputs.  On Pixhawk boards this parameter only affects AUX pins.  Write the parameter 
and reboot the autopilot.

> Now the RPM library must be enabled. In the following sections, we will use the second instance of RPM sensor for parameter examples.

Reviewer: Not an issue, but just wondering why we are doing an example on the 2nd instance and not the first?

Author: just to highlite that there are more than one RPM instance available

> Set the parameter :ref:`RPM2_TYPE<RPM2_TYPE>` to 1 for a GPIO pin based sensor.  Write the parameter to ArduPilot then refresh/fetch the 

Reviewer: Can the example set RPM2_TYPE to 2 please.  We have left type = 1 active so as not to break any existing setups but the param meta-data does now say "1:Not Used" so we should maintain the theme in the wiki.

Author: done

For clarification, this is not the same as the RPM that can be passed 
via serial telemetry with ESCs.  For information on how to set up RPM reporting with capable ESCs, see the :ref:`ESC Telemetry<esc-telemetry>`.
For information on how to set up RPM logging with BL Heli see the :ref:`BLHeli Telemetry<common-dshot-blheli32-telemetry>`.

The setup for electrical commutation RPM sensors is much the same as hall effect sensors, so the steps above are applicable.  The only difference is the scaling value 
to be entered in the :ref:`RP2_SCALING<RPM2_SCALING>` parameter.  Now, the scaling value is a function of the number of poles in the motor and should be the reciprocal of the number of 
poles.  E.g. A 4 pole motor will need a scaling value of 0.25.

Optical Sensors
===============

Again, the setup of optical sensors is much the same as the hall effect sensor.  The only difference is the scaling value.  :ref:`RPM2_SCALING<RPM2_SCALING>` should be set to be the reciprocal 
of the number of reflective obstacles that pass the sensor in a single rotation.

Harmonic Notch Center Frequency
===============================

> If ``RPMx_TYPE`` = 4, then the center frequency of the harmonic notch, however it is obtained (See :ref:`common-imu-notch-filtering`), is reported.

Reviewer: Maybe remove the sub-clause and move it to the end of the sentence?  Took me a couple of goes reading it to understand what was being said. e.g:
```suggestion
If ``RPMx_TYPE`` = 4, then the center frequency of the harmonic notch is reported.  See :ref:`common-imu-notch-filtering` for how this value is obtained.
```
